engine-tests/src/test/java/org/terasology/documentation/BindingScraper.java

```diff
@@ -18,6 +18,7 @@
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.Multimap;
 import org.terasology.engine.core.module.ModuleManager;
+import org.terasology.engine.core.module.ModuleManagerImpl;
```

Comment:
why are there a bunch of these changes adding an import but not anything that would use that import?